.golangci.yml

```diff
@@ -1,7 +1,7 @@
 linters:
   enable:
     - asciicheck
-    - bodyclose
+    # - bodyclose // to be fixed by golangci-lint
```

Comment:
bodyclose support is supposed to be re-enabled as part of golangci-lint v1.47.3, according to https://github.com/golangci/golangci-lint/pull/3001.